cms/static/coffee/spec/views/course_info_spec.coffee

```diff
@@ -232,6 +232,20 @@ define ["js/views/course_info_handout", "js/views/course_info_update", "js/model
 
                 @handoutsEdit.render()
 
+            it "saves <ol></ol> when content left empty", ->
+                requests = AjaxHelpers["requests"](this)
+
+                # Enter empty string in the handouts section, verifying that the model
+                # is saved with '<ol></ol>' instead of the empty string
```

Comment:
kept the convention of commenting consistent throughout this file so I used this instead of a multi line comment.